.github/workflows/py-dependency-check.yml

```diff
@@ -20,7 +20,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [single-cell-8c64g-runner, macos-latest]
```

Comment:
Why does the runner appear in the `os` field?

Reply:
if you look a few lines below, these are the labels we provide to `runs-on`, which accepts a runner.  I suppose we could rename `os` to something better (e.g., `runner`?), but using `os` has been our convention for a while

Reply:
I see it now